recipes/datastax.rb

```diff
@@ -35,8 +35,8 @@
   action :install
 end
 
-package node['cassandra']['dsc_version'] do
-  action :install
+execute "install_cassandra" do
+  command "apt-get install -y #{node[:cassandra][:dsc_version]}=#{node[:cassandra][:version]}* cassandra=#{node[:cassandra][:version]}"
```

Comment:
Found this bug in the cookbook and fixed it. Though it doesn't have any relevance at this point of time, it could be of help in the future.